src/Test_NavEspn.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Firefox 35
+ * selenium-server-standalone-2.44.0.jar
+ * 
+ * Note I am still integration Saucelab
+ * I am not using Maven which would import dependencies used by Saucelab
+ * that will come next.
+ * I am using Ant which I wanted to try
+ */
+import java.util.concurrent.TimeUnit;
+
+import org.junit.*;
+
+import static org.junit.Assert.*;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.remote.CapabilityType;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+//import com.saucelabs.junit.Parallelized;
+//import com.saucelabs.junit.ConcurrentParameterized;
+//import com.saucelabs.junit.SauceOnDemandTestWatcher;
+
+public class Test_NavEspn {
+	  private WebDriver driver;
+	  private String baseUrl;
+	  private StringBuffer verificationErrors = new StringBuffer();
+	  private String browserName = "firefox"; //firefox, chrome, internet explorer, safari, opera, iPad, iPhone, android
+	  private String osName =  "MAC";          // MAC, WIN8, XP, WINDOWS, ANY, ANDROID
+//	  public SauceOnDemandAuthentication authentication = new SauceOnDemandAuthentication("rduvalXXXX", "XXX87483-1bfd-4635-87a6-7b90546396c9");
+
+	  @Before
+	  public void setUp() throws Exception {
+	/* Links on capabilities
+	 * https://code.google.com/p/selenium/wiki/DesiredCapabilities
+	 * http://www.browserstack.com/automate/capabilities
+	 */
+		  
+       DesiredCapabilities capabilities = new DesiredCapabilities();
+       capabilities.setCapability(CapabilityType.BROWSER_NAME, browserName );
+       capabilities.setCapability(CapabilityType.PLATFORM, osName);
+       capabilities.setCapability("name", "Test_NavEspn");
+       driver = new FirefoxDriver();
+       baseUrl = "http://espn.go.com/";
+       driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
+	  }
+
+	  @Test
+	  public void testEspn() throws Exception {
+	    driver.get(baseUrl);
```

Comment:
No, I was following a requirement for an exercise using my know methods.  Thanks for the input, I will pursue this.  It may enhance a beta frame work I created else where.


> On Feb 26, 2015, at 12:59 PM, Brian R. Jackson <notifications@github.com> wrote:
> 
> In src/Test_NavEspn.java <https://github.com/jaxzin/fizzbuzz-selenium/pull/32#discussion_r25463688>:
> 
> > +	 * https://code.google.com/p/selenium/wiki/DesiredCapabilities
> > +	 * http://www.browserstack.com/automate/capabilities
> > +	 */
> > +		  
> > +       DesiredCapabilities capabilities = new DesiredCapabilities();
> > +       capabilities.setCapability(CapabilityType.BROWSER_NAME, browserName );
> > +       capabilities.setCapability(CapabilityType.PLATFORM, osName);
> > +       capabilities.setCapability("name", "Test_NavEspn");
> > +       driver = new FirefoxDriver();
> > +       baseUrl = "http://espn.go.com/";
> > +       driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
> > +	  }
> > +
> > +	  @Test
> > +	  public void testEspn() throws Exception {
> > +	    driver.get(baseUrl);
> Did you consider using a Page Object pattern here?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/jaxzin/fizzbuzz-selenium/pull/32/files#r25463688>.
>